From b1e7f2cd0074f0ae32139faca3f0b8c2db2f2458 Mon Sep 17 00:00:00 2001
From: Rintaro Ishizaki <fs.output@gmail.com>
Date: Wed, 4 May 2016 16:59:21 +0900
Subject: [PATCH] [build-script][SR-1287] Revert "Clear CMakeCache.txt before
 (re)configuration"

https://cmake.org/runningcmake/
> If you have not already hand-edited the cache file,
> you could just delete it before re-running CMake.

is not quite right.
see: https://cmake.org/Bug/view.php?id=14820
---
 utils/build-script-impl | 1 -
 1 file changed, 1 deletion(-)

diff --git a/utils/build-script-impl b/utils/build-script-impl
index 766967f..42faf39 100755
--- a/utils/build-script-impl
+++ b/utils/build-script-impl
@@ -1963,7 +1963,6 @@ for deployment_target in "${HOST_TARGET}" "${CROSS_TOOLS_DEPLOYMENT_TARGETS[@]}"
                     ( ! -z "${generator_output_path}" && ! -f "${generator_output_path}" ) ]] ; then
             set -x
             mkdir -p "${build_dir}"
-            rm -f "${cmake_cache_path}"
             (cd "${build_dir}" && "${CMAKE}" "${cmake_options[@]}" "${EXTRA_CMAKE_OPTIONS[@]}" "${source_dir}")
             { set +x; } 2>/dev/null
         fi
-- 
2.8.2
